code/game/objects/items/weapons/storage/holster.dm

```diff
@@ -75,7 +75,7 @@
 	sound_in = 'sound/effects/holsterin.ogg'
 	sound_out = 'sound/effects/holsterout.ogg'
 
-	w_class = ITEM_SIZE_NORMAL
+	w_class = ITEM_SIZE_BULKY
```

Comment:
Consider using `slot_flags = SLOT_BELT | SLOT_DENYPOCKET` instead, holster/sheath can't be that chonky, large pouch would fit at least three times as much content and it's normal-sized.

Reply:
k